convert/convert.go

```diff
@@ -226,15 +226,14 @@ func getWigChromIndex(s string, wigSlice []*wig.Wig) int {
 	return -1
 }
 
-//PairwiseFaToVcf takes in a pairwise multiFa alignment and returns Vcf entries for segregating sites with the first entry as the reference and the second fasta entry as the alt allele.
+//PairwiseFaToVcf takes in a pairwise multiFa alignment and writes Vcf entries for segregating sites with the first entry as the reference and the second fasta entry as the alt allele.
 //This will have to be done by chromosome, as a pairwise multiFa will only have two entries, thus containing one chromosome per file.
-func PairwiseFaToVcf(f []*fasta.Fasta, chr string) []*vcf.Vcf {
+func PairwiseFaToVcf(f []*fasta.Fasta, chr string, out fileio.EasyWriter, substitutionsOnly bool, retainN bool) {
```

Comment:
Based on my previous comment, this would need a change from fileio.EasyWriter to *fileio.EasyWriter